src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,17 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
 public class Main {
     public static void main(String[] args) {
-
+        Injector injector = Injector.getInstance("mate");
```

Comment:
`Injector` injector = Injector.getInstance("mate.academy");`

Reply:
It is the same, because 'mate' package is parent of 'academy' package

Reply:
ok)